CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+* An `include_rgb` option has been added to `rtc_sentinel.py` to include an RGB decomoposition GeoTIFF in the output
```

Comment:
should probably write this like "An `--include-rgb` option..." since you're talking about the entrypoint, might be worth  calling out the funtion parameter as well. Something like:

```
* Ability to include an RGB deompostion GeoTIFF in the output
  * an `--include-rgb` option has been added to `rtc_sentinel.py`
  * an `include_rgb` keyword argument has been added to `hyp3_gamma.rtc.rtc_sentinel.rtc_sentinel_gamma`
```